cli/updates/types.go

```diff
@@ -0,0 +1,8 @@
+package updates
+
+// Update stores available updates for rendering in a table
+type Update struct {
```

Comment:
This can now be remove (https://github.com/docker/cli/pull/1260/files#diff-2349949258e8913f2e5141f6d989e2bdR144)